packages/eslint-plugin/src/rules/return-await.ts

```diff
@@ -307,13 +307,14 @@ export default createRule({
 
       if (!isThenable) {
         if (isAwait) {
-          // any/unknown could be thenable; do not auto-fix
-          const useAutoFix = !(isTypeAnyType(type) || isTypeUnknownType(type));
-
+          // sync behavior with return-await
+          if (isTypeAnyType(type) || isTypeUnknownType(type)) {
+            return;
+          }
           context.report({
             messageId: 'nonPromiseAwait',
             node,
-            ...fixOrSuggest(useAutoFix, {
+            ...fixOrSuggest(true, {
               messageId: 'nonPromiseAwait',
               fix: fixer => removeAwait(fixer, node),
             }),
```

Comment:
[Refactor] Since we don't report suggestions anymore, we don't need to use `fixOrSuggest` here. How about replacing the whole `...fixOrSuggest(...)` call with just `fix: fixer => removeAwait(fixer, node),`?

Reply:
Thanks for reviewing it, and i fixed this code the way you showed me.